src/cli/lambdapi.ml

```diff
@@ -63,7 +63,11 @@ let websearch_cmd cfg rules port require header_file dbpath_opt =
         query language specification</a> to learn about the query language.
         <br>The query language also uses the <a
         href=\"https://lambdapi.readthedocs.io/en/latest/terms.html\">
-        Lambdapi terms syntax</a>.<br>
+        Lambdapi terms syntax</a>.<br> with the possibility to use,
+        for commodity,
+        \"forall\" and \"->\"  instead of \"Π\" and \"→\" respectively
+        (note,results are displayed with the Unicode symbols
+        \"Π\" and \"→\").
```

Comment:
Why not replacing Π and → by forall and -> in the output too?
Actually, there should be an option `--coq-syntax` for enabling this replacement.

Reply:
This replacement should be applied to the header as well.